tools/bench_deepgemm_stream.py

```
@@ -0,0 +1,426 @@
+Usage:
+  python tools/bench_deepgemm_stream.py
+  python tools/bench_deepgemm_stream.py --m_list 64 128 512 1024 4096
+  python tools/bench_deepgemm_stream.py --profile   # NVTX + nsys
+
+  # Full nsys trace:
+  nsys profile --trace=cuda,nvtx -o deepgemm_stream --force-overwrite true \\
+      python tools/bench_deepgemm_stream.py --profile --warmup 5 --iters 20
+"""
+
+import argparse
+import os
```

Flake8 (7.3.0)

[error] 'os' imported but unused

(F401)